packages/lit-task-auto-top-up/src/constants.ts

```diff
@@ -1,3 +1,3 @@
 export const DEFAULT_RECIPIENT_LIST_URL =
-  'https://raw.githubusercontent.com/LIT-Protocol/lit-scheduled-tasks/blob/main/worker/recipient_list.json';
+  'https://raw.githubusercontent.com/LIT-Protocol/lit-scheduled-tasks/blob/main/worker/recipient_list_habanero.json';
```

Comment:
I can't seem to access this .json file though, do we need the `/blob` path? If I remove it I can see the content:
https://raw.githubusercontent.com/LIT-Protocol/lit-scheduled-tasks/main/worker/recipient_list_habanero.json

Reply:
Good catch -- I think the `/blob` was from a prior URL that was pointing to an explicit commit. Since this is set explicitly via environment variable in our workers, it wasn't actually being used anywhere so I missed it.  All set; I updated it to match reality.